plugins/inputs/azure_storage_queue/azure_storage_queue.go

```diff
@@ -99,19 +99,19 @@ func (a *AzureStorageQueue) Gather(acc telegraf.Accumulator) error {
 		marker = queuesSegment.NextMarker
 
 		for _, queueItem := range queuesSegment.QueueItems {
-			a.Log.Debugf("Processing queue '%s' of storage account '%s'", queueItem.Name, a.StorageAccountName)
+			a.Log.Debugf("Processing queue %q of storage account %q", queueItem.Name, a.StorageAccountName)
 			queueURL := serviceURL.NewQueueURL(queueItem.Name)
 			properties, err := queueURL.GetProperties(ctx)
 			if err != nil {
-				a.Log.Errorf("Error getting properties for queue %s: %s", queueItem.Name, err.Error())
+				a.Log.Errorf("Error getting properties for queue %s: %w", queueItem.Name, err)
 				continue
 			}
 			var peekedMessage *azqueue.PeekedMessage
 			if a.PeekOldestMessageAge {
 				messagesURL := queueURL.NewMessagesURL()
 				messagesResponse, err := messagesURL.Peek(ctx, 1)
 				if err != nil {
-					a.Log.Errorf("Error peeking queue %s: %s", queueItem.Name, err.Error())
+					a.Log.Errorf("Error peeking queue %s: %w", queueItem.Name, err)
```

Comment:
`%w` should be only used for creating error, not for printing. Use `%s` or `%v` instead.